README.md

```diff
@@ -79,6 +79,56 @@ cd corral
 
 You can make a debug build with `.\make.ps1 build -Config Debug`
 
+## Getting started using Corral
+
+After instllation, make sure Corral's current path is also set in your PATH environment variable. This will make it easier by not entering the full path each time when you use one of Corral commands in your projects.
+
+Follow these steps to create your first project using Corral.
+
+1. Create an empty folder and switch to this directory. This will be our example project to use Corral
+
+```bash
+mkdir myproject
+cd myproject
+```
+
+2. Initialize Corral. It will create `corral.json` and `lock.json` files. At this moment they won't have much information since you haven't added any dependency yet.
+
+```bash
+corral init
+```
+
+3. Add a dependency. This is the way to tell Corral that your project depends on this and you want to include it when building your project.
+
+```bash
+corral add github.com/jemc/pony-inspect.git
+```
+
+4. Use that dependency in your code. Create a file `main.pony` with following code.
```

Comment:
If the comment I made previously on short description followed by long description for "Create a project. Make an empty folder..." is followed then I think this should become "Use a dependency. Create a file..." so removing "in your code." from the short description to make it shorter.